.github/actions/ci-terraform-validate/action.yml

```diff
@@ -65,7 +65,9 @@ runs:
         quiet: true
         framework: terraform
         output_format: cli
-        skip_check: CKV_AWS_119,CKV_AWS_144,CKV_AWS_145,CKV_AWS_158,CKV_AWS_191,CKV_AWS_336,CKV_AWS_337,CKV_AWS_378,CKV2_AWS_34,CKV2_AWS_62,CKV_TF_1,CKV_TF_3
+        skip_check: CKV_AWS_7,CKV_AWS_18,CKV_AWS_51,CKV_AWS_65,CKV_AWS_68,CKV_AWS_119,CKV_AWS_136,CKV_AWS_144,CKV_AWS_145,CKV_AWS_149,CKV_AWS_158,CKV_AWS_159,CKV_AWS_163,CKV_AWS_191,CKV_AWS_336,CKV_AWS_337,CKV_AWS_378,CKV2_AWS_34,CKV2_AWS_62,CKV_TF_1,CKV_TF_3
```

Comment:
Přidal jsem do ignore pravidla jako access logging 3 bucketů, rotaci KMS klíčů apod. (takový ty na který ještě nejsme dost velcí 😄 ale chci aby bylo explicitně jasný že o nich víme). Píšu si do backlogu úkol to někam, asi do Confluence, popsat který to jsou a proč.

Reply:
Tady se "centrálně" ignorují pravidla který ignorujeme plošně všude. Ty `#checkov:skip=` komentáře (jako třeba https://github.com/FigurePOS/fgr-service-emailing/blob/9b862345820b486f780288882cc881ab7f0b08c4/tf/sqs.tf#L15) se snažím používat jen když jde o nějakou výjimku.